praw/models/reddit/wikipage.py

```diff
@@ -162,12 +380,43 @@ def mod(self) -> WikiPageModeration:
         """
         return WikiPageModeration(self)
 
+    @property
+    def revision_hidden(self) -> Union[bool, None]:
+        """Check the ``revision_hidden`` attribute of a WikiPage revision.
+
+        Example usage:
+
+        .. code-block:: python
+
+            one_revision = reddit.subreddit("test").wiki["praw"].revisions(limit=1)
+            wiki_page = next(one_revision).get("page")
+            wiki_page.revision_hidden
+
+        To check and set the ``revision_hidden`` attribute of a WikiPage, try:
+
+        .. code-block:: python
+
+            wiki_page = reddit.subreddit("test").wiki["praw"].revision("[ID]")
+            wiki_page.revision_hidden = wiki_page.mod._check_status()
+
+        This does not change the visibility of the revision on Reddit.
```

Comment:
Could we incorporate the logic of ``wiki_page.mod._check_status()`` into this property instead of recommending the user to utilize a private method?

Reply:
I'd rather keep the check and set separate, since `toggle_visibility` doesn't need a check. I could add a public method that checks and sets the `revision_hidden` attribute, or I could make `_check_status` public.